controllers/elasticSearchController.js

```diff
@@ -2,40 +2,67 @@ var express = require('express');
 var elasticService = require('../services/elasticService.js');
 var logger = require('../utils/logger');
 
+
+var extractLog = function(log, i) {
```

Comment:
Awesome. I like this improvement over doing all the extraction in the jade/pug file.